tests/algorithms/test_gated_linear_units.py

```diff
@@ -73,16 +73,27 @@ def assert_is_glu_instance(model):
     ), 'composer.algorithms.gated_linear_units.gated_linear_unit_layers.BERTGatedFFOutput is not found in the post-surgery model.'
 
 
-def test_gated_linear_units_functional(synthetic_bert_state: Tuple):
-    state, _, _ = synthetic_bert_state
+@pytest.mark.parametrize('synthetic_state_family', ['bert', 'simple_transformer'])
+def test_gated_linear_units_functional(synthetic_state_family: str, request: pytest.FixtureRequest):
+    if synthetic_state_family == 'simple_transformer':
```

Comment:
the xfail can be put in the parametrization like `pytest.param('simple_transformer', marks=pytest.mark.xfail)`